packages/code/src/CodeContext.tsx

```diff
@@ -0,0 +1,20 @@
+import { createContext, useContext } from 'react';
+import { Theme } from '@leafygreen-ui/lib';
+
+interface CodeData {
+  theme: Theme;
+  darkMode: boolean;
+}
+
+export function useCodeContext() {
+  const { theme, darkMode } = useContext(CodeContext);
+
+  return { theme, darkMode };
+}
+
+export const CodeContext = createContext<CodeData>({
+  theme: Theme.Light,
+  darkMode: false,
+});
```

Comment:
Could we use a new `LeafyGreenProvider` instead, or do you anticipate needing other properties in the future?

Reply:
I don't anticipate anything else. I'll try it out!